package.json

```diff
@@ -34,7 +36,7 @@
     "@types/react": "^17.0.5",
     "@types/react-dom": "^17.0.4",
     "babel-loader": "^8.2.2",
-    "npm-run-all": "^1.7.0",
+    "concurrently": "^6.2.0",
```

Comment:
why change to concurrently when `npm-run-all` was chosen?

Reply:
They are similar, but last commit of `npm-run-all` was almost 3 years ago, last commit of `concurrently` 5 days ago.
Anyway, we can use any of them, no strong preference.

Reply:
fair enough, let's keep it